### PR TITLE
Feature/create context

### PR DESCRIPTION
**Pull request by kavakai**
### What (if any) features are you implementing?
- Create Context files for Journal context and quote context. 
- Build initial functionality creating journal entry. 

### What (if anything) did you refactor?
-Cleaned up unused code, and console.logs
### Were there any issues that arose?
- Not really.

### Refactor checklist
- [x] Check variable and parameter names: are they descriptive and unique?
- [x] Is the code DRY? SRP?
- [x] Is it easy for others to read your code? (explanatory comments needed?)
- [x] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [x] Are commit messages formatted correctly?

### CODE REVIEW

**Review by gjtrecny — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -11,17 +13,21 @@ import './reset.scss'
...
-        <Route exact path='/' component={LandingPage}/>
-        <Route exact path='/features' component={FeaturesPage}/>
-        <Route exact path='/journal' component={JournalPage} />
-        <Route exact path='/mood' component={MoodPage} />
-        <Route exact path='/dashboard' component={Dashboard} />
-        <Route exact path='/error' component={ErrorPage} />
-        <Redirect to="/error" />
-     </Switch>
-    </div>
+    <QuoteProvider>
```
Comment (anewb87) [inline src/App.js:16]: The providers are working. The providers are working.

File: src/Contexts/QuoteContext.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { useState, useEffect, createContext } from "react";
```
Comment (anewb87) [inline src/Contexts/QuoteContext.js:1]: Our first context

File: src/Utilities/Date.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export const createDate = () => {
```
Comment (anewb87) [inline src/Utilities/Date.js:1]: Look Robbie! A utilities folder with a file!

**Review by tommi-t-nguyen — APPROVED** [review]
👍

**Review by anewb87 — APPROVED** [review]
Making some moves on the GSM front.

File: src/Utilities/Date.js (hunk 3 above)
Comment (robbiejaeger) [reply src/Utilities/Date.js:1]: 🎉🎉🎉🎉🎉

**Pull request merged**